tests/test_dimension.py

```diff
@@ -210,6 +210,21 @@ def test_modulo_dims_generation_v2(self):
         assert np.all(f.data[3] == 2)
         assert np.all(f.data[4] == 4)
 
+    def test_degenerate_to_zero(self):
```

Comment:
Docstring or a comment explaining what this does would help understanding at a glance

Reply:
adding docstring